.github/workflows/run-k8s-tests-on-tdx.yaml

```diff
@@ -0,0 +1,50 @@
+name: CI | Run kubernetes tests on TDX
+on:
+  workflow_call:
+    inputs:
+      registry:
+        required: true
+        type: string
+      repo:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+
+jobs:
+  run-k8s-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        vmm:
+          - qemu-tdx
+    runs-on: tdx
```

Comment:
Do you think in protect that machine by allowing jobs only with a certain label like "ok-to-test"?

Reply:
I've been going back and forth on this, and the PRs will only automatically run for members of the Kata Containers organisation, which are the same ones that can add an "ok-to-test" label.

With this in mind, I kinda prefer to trust that no malicious attempt will come from inside the Kata Containers org.